packages/analytics/amplify_analytics_pinpoint/pubspec.yaml

```diff
@@ -9,6 +9,8 @@ environment:
 
 dependencies:
```

Comment:
I can't leave a comment on the correct line, but on line 4 the homepage will need to be updated.

```diff
- https://github.com/aws-amplify/amplify-flutter/tree/main/packages/amplify_analytics_pinpoint
+ https://github.com/aws-amplify/amplify-flutter/tree/main/packages/analytics/amplify_analytics_pinpoint
```

This will unfortunately make pub analysis fail temporarily because the new URL will return a 404. Once the PR is merged, the URL will return a 200 and pub analysis will pass. The is the change @haverchuck made in #1424 for auth.

We will just have to merge this PR with one failing CI step. We should confirm that the homepage change is the only reason it is failing though.

Reply:
This comment applies to the storage PR as well by the way.

Both analytics and storage are scoring 110/120 on their pub score, which is passing in CI. With the homepage change, they will start scoring 100/120, which will fail in CI. If they score below 100, then we should figure out why before merging.

We should also keep an eye on the post-merge CI run to confirm it passes after this merges.

Reply:
Thanks sounds good, updated homepage field in pubspec.yaml